pyriemann_qiskit/utils/distance.py

```diff
@@ -60,25 +82,61 @@ def log_prod(m1, m2):
 
     result = optimizer.solve(prob, reshape=False)
 
-    return 1 - result
+    # compute distance
+    _2VecLogYD = 2 * sum(result[i] * log_prod(B, A[i]) for i in matrices)
+    wtDw = sum(
+        result[i] * result[j] * log_prod(A[i], A[j]) for i in matrices for j in matrices
+    )
+    distance = wtDw - _2VecLogYD
```

Comment:
After
```
from pyriemann.utils.distance import distance_logeuclid
from pyriemann.utils.mean import mean_logeuclid
```
you can simply do
```suggestion
    # compute nearest matrix and distance
    C = mean_logeuclid(A, w)
    distance = distance_logeuclid(C, B)
```

Reply:
Ok, excellent point :)